lib/stdlib/select.juttle

```diff
@@ -59,3 +59,28 @@ export sub percentile(field, p, by=[]) {
 export sub median(field, by=[]) {
     percentile -field field -p 0.5 -by by
 }
+
+//
+// forward the top N points ordered by the `by` field provided
+//
+// Parameters:
+//
+//   * n: number of points to forward
+//   * by: (opt) field to sort by
+//
+export sub top(n, by) { 
+    sort by | tail n 
```

Comment:
This would be clearer with explicit `sort -asc`, which I know is default behavior, but if we spell it out then having `tail` not `head` makes sense